src/views/private/admin/components/showProducts/components/editProduct/index.js

```diff
@@ -31,8 +31,7 @@ export default function EditProduct() {
     const dispatch = useDispatch();
     const data = useSelector(state => state?.categoryReducer?.payload)
     const productData = useSelector(state => state?.singleProductReducer?.payload)
-    const {id} = useParams();
-    console.log(id)
+    const { id } = useParams();
```

Comment:
you can give specific name to use in component .
example:
const {id : productId} = useParams();
ypu get the req value with name "productId"